src/main/java/org/folio/consortia/domain/entity/PublicationStatusEntity.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.consortia.domain.entity;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.springframework.data.annotation.CreatedDate;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+@Table(name = "pc_state")
+public class PublicationStatusEntity {
```

Comment:
Seems I forgot to mention in story AC, please extend with AuditableEntity to both tables to have full metadata info. 
We will use this approach for all tables in mod-consortia.
Related PR: https://github.com/folio-org/mod-consortia/pull/45/files